app/views/handlers/individual_response.py

```diff
@@ -491,13 +512,109 @@ def handle_post(self):
             self._update_section_status(CompletionStatus.INDIVIDUAL_RESPONSE_REQUESTED)
             return redirect(
                 url_for(
-                    "individual_response.get_individual_response_post_address_confirmation"
+                    "individual_response.get_individual_response_post_address_confirmation",
+                    return_to=self._return_to,
                 )
             )
 
         return redirect(
             url_for(
                 "individual_response.get_individual_response_how",
                 list_item_id=self._list_item_id,
+                return_to=self._return_to,
+            )
+        )
+
+
+class IndividualResponseWhoHandler(IndividualResponseHandler):
+    def __init__(self, schema, questionnaire_store, language, request_args, form_data):
+        self._list_name = schema.json.get("individual_response", {}).get("for_list")
```

Comment:
As we are calling schema.json.get("individual_response", {}) a few times now, would it be best to have its own method in QuestionnaireSchema?